core/templates/dev/head/directives.js

```diff
@@ -157,18 +157,16 @@ oppia.directive('mobileFriendlyTooltip', ['$timeout', function($timeout) {
   };
 }]);
 
-oppia.directive( 'updateLanguage', updateLanguage );
-function updateLanguage( $rootScope ) {
-    return {
-      link: function( scope, element ) {
-        var listener = function( event, translationResp ) {
-          var defaultLang = "en",
-              currentlang = translationResp.language;
-
-          element.attr("lang", currentlang || defaultLang );
-        };
+oppia.directive('updateLanguage', [function($rootScope) {
```

Comment:
Still not correct. Please look carefully at other directives in this file and follow the existing code style. I'm seeing issues with indentation (line 166), spacing before and after parens, not specifying the dependencies as strings (which means this is going to break on minification), and single vs double quotes.